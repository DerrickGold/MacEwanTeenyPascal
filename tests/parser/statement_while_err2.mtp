begin

	(* while statement: missing condition *)
	while do
		sample := sample + 1
end
.