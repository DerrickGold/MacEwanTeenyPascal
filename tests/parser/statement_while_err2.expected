Syntax Error: line 4:
Unexpected token (TOK_KEY_DO, 'do'), likely missing a(n)  
  'TOK_LPAREN' or
  'TOK_KEY_NOT' or
Possibly due to malformed or missing:
  Non-terminal: 'While Statement' or
  Non-terminal: 'Condition'
